app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,32 @@
+class TicketsController < ApplicationController
+  def new
+    @ticket = list.tickets.new
+  end
+
+  def create
+    @ticket = list.lickets.new(ticket_params)
+    if @ticket.save
+      redirect_to board_path(board)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def board
+    @board ||= Board.find(params[:board_id])
```

Comment:
board jest zbedny w tej sciezce... przeciez mozesz go wyciagnac majac liste: list.board to po co wwalac sobie dodatkowy parametr w url'u